content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -189,12 +194,14 @@ that originate from outside your cluster.
 
 ## Resources
 The flow control API involves two kinds of resources.
-[PriorityLevelConfigurations](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#prioritylevelconfiguration-v1alpha1-flowcontrol-apiserver-k8s-io)
+[PriorityLevelConfigurations](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#prioritylevelconfiguration-v1beta1-flowcontrol-apiserver-k8s-io)
```

Comment:
nit: This could be a list?

The flow control API involves two types of resources:

* PriorityLevelConfigurations .... define the available isolation classes and the share of the available concurrency
budget that each can handle, and allow for fine-tuning queuing behavior.
* FlowSchemas ...